Dev/VideoChatClient/Input/ImageInput.cpp

```diff
@@ -0,0 +1,27 @@
+#include "ImageInput.hpp"
+
+#include "VideoChatClientException/VideoChatClientException.hpp"
+
+ImageInput::ImageInput() : m_capture()
+{
+	m_capture.open(cv::CAP_ANY);
+	if (!m_capture.isOpened())
+	{
+		throw VideoChatClientException(VideoChatClientStatus::IMAGE_INPUT_OPEN_CAPTURE_FAILED, 0); //TODO
+	}
+	m_capture.set(cv::CAP_PROP_FRAME_WIDTH, IMAGE_WIDTH);
```

Comment:
I see you made the width&height as consts, maybe we would like later to get the screen or windows resolution and change according to that. just a note...